### PR TITLE
Run Fusion.js verification on all PRs

### PR DESCRIPTION
**Pull request by alxmyth**
As discussed in sprint planning, the proposed change will run our Fusion.js verification script on all pull requests.  Note that non-release PRs will be non-blocking.

### CODE REVIEW

File: src/release-verification.js [hunk 1]
```diff
@@ -75,7 +67,8 @@ module.exports = robot => {
         'release-pr-head-sha': pr.head.sha,
         'release-pr-head-repo-full-name': pr.head.repo.full_name,
         'release-pr-base-repo-full-name': pr.base.repo.full_name,
-        'release-pr-prerelease': String(prerelease),
+        'release-pr-prerelease': String(isPrerelease),
```
Comment (alxmyth) [inline src/release-verification.js:70]: Do we use this anywhere?  A [quick search](https://github.com/search?q=release-pr-prerelease+org%3Afusionjs+org%3Auber-web+org%3Auber-workflow&type=Code) doesn't turn up anything.

**Comment by alxmyth** [comment]
May need to upgrade failing snapshot tests due to this change.

File: src/__tests__/__snapshots__/buildkite-meta-data.test.js.snap [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 
-exports[`buildkite meta-data populates meta-data with branch information 1`] = `"{\\"commit\\":\\"HEAD\\",\\"branch\\":\\"master\\",\\"message\\":\\"probot-playground, Release v1.0.7 - release verification\\",\\"author\\":{\\"name\\":\\"KevinGrandon\\"},\\"meta_data\\":{\\"release-pr-number\\":\\"6\\",\\"release-pr-head-sha\\":\\"23999b445a8daf81238dbcdc08776f4489f4aa98\\",\\"release-pr-head-repo-full-name\\":\\"KevinGrandon/probot-playground\\",\\"release-pr-base-repo-full-name\\":\\"KevinGrandon/probot-playground\\",\\"release-pr-prerelease\\":\\"false\\"}}"`;
+exports[`buildkite meta-data populates meta-data with branch information 1`] = `"{\\"commit\\":\\"HEAD\\",\\"branch\\":\\"master\\",\\"message\\":\\"probot-playground, Release v1.0.7 - release verification\\",\\"author\\":{\\"name\\":\\"KevinGrandon\\"},\\"meta_data\\":{\\"release-pr-number\\":\\"6\\",\\"release-pr-head-sha\\":\\"23999b445a8daf81238dbcdc08776f4489f4aa98\\",\\"release-pr-head-repo-full-name\\":\\"KevinGrandon/probot-playground\\",\\"release-pr-base-repo-full-name\\":\\"KevinGrandon/probot-playground\\",\\"release-pr-prerelease\\":\\"false\\",\\"is-release-pr\\":\\"true\\"}}"`;
```
Comment (alxmyth) [inline src/__tests__/__snapshots__/buildkite-meta-data.test.js.snap:3]: Adds "`"is-release-pr": "true"` to `meta_data` here.

**Review by ganemone — APPROVED** [review]

**Comment by alxmyth** [comment]
!merge

**Pull request merged**